integration/config/core.yaml

```diff
@@ -382,8 +394,6 @@ peer:
     #   escc:
     #     name: DefaultESCC
     #     library: /etc/hyperledger/fabric/plugin/escc.so
-    #
-    #TODO: Enable FPC Additions once tlcc and plugins are intergrated
```

Comment:
todo complete?

Reply:
Nope, only part of it (tlcc) and  moved rest further down & closer to where the corresponding additions to be uncommented are